src/main/java/me/totalfreedom/totalfreedommod/command/Command_list.java

```diff
@@ -2,27 +2,25 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import me.totalfreedom.totalfreedommod.config.ConfigEntry;
+import me.totalfreedom.totalfreedommod.player.FPlayer;
 import me.totalfreedom.totalfreedommod.rank.Displayable;
 import me.totalfreedom.totalfreedommod.rank.Rank;
-import me.totalfreedom.totalfreedommod.util.FUtil;
 import org.apache.commons.lang3.StringUtils;
 import org.bukkit.ChatColor;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.IMPOSTOR, source = SourceType.BOTH)
-@CommandParameters(description = "Lists the real names of all online players.", usage = "/<command> [-a | -i | -f]", aliases = "who")
+@CommandParameters(description = "Lists the real names of all online players.", usage = "/<command> [-a | -i | -v]", aliases = "who")
 public class Command_list extends FreedomCommand
 {
 
     private static enum ListFilter
     {
-
         PLAYERS,
         ADMINS,
-        FAMOUS_PLAYERS,
```

Comment:
Why remove famous players? While I didnt know it was a thing, im not sure why we're removing the functionality?